src/test/java/seedu/sellsavvy/logic/commands/ordercommands/MarkOrderCommandTest.java

```diff
@@ -0,0 +1,96 @@
+package seedu.sellsavvy.logic.commands.ordercommands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.sellsavvy.logic.commands.ordercommands.OrderCommandTestUtil.assertCommandFailure;
+import static seedu.sellsavvy.logic.commands.ordercommands.OrderCommandTestUtil.assertCommandSuccess;
+import static seedu.sellsavvy.testutil.TypicalIndexes.INDEX_FIRST_ORDER;
+import static seedu.sellsavvy.testutil.TypicalIndexes.INDEX_SECOND_ORDER;
+import static seedu.sellsavvy.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.Messages;
+import seedu.sellsavvy.model.Model;
+import seedu.sellsavvy.model.ModelManager;
+import seedu.sellsavvy.model.UserPrefs;
+import seedu.sellsavvy.model.order.Order;
+import seedu.sellsavvy.model.person.Person;
+
+public class MarkOrderCommandTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs()).createCopy();
+        Person personToMarkOrderUnder = model.getFilteredPersonList().get(3);
+        model.updateSelectedPerson(personToMarkOrderUnder);
+    }
+
+    @Test
+    public void execute_validIndexOrderList_success() {
+        MarkOrderCommand markOrderCommand = new MarkOrderCommand(INDEX_FIRST_ORDER);
+
+        Model expectedModel = model.createCopy();
+        Order orderToBeMarked = expectedModel.getFilteredPersonList().get(3)
```

Comment:
maybe you could standardise and put INDEX_FOUNRTH_PERSON.getZeroBased?

Reply:
will change by creating a new integer constant for `MarkOrderCommandTest` called `TARGET_INDEX`

Reply:
shouldn't it be in TypicalIndex? In fact Sutong already created it in previous commit

Reply:
oh my bad, changing to `INDEX_FOURTH_PERSON.getZeroBased()`